test/spec/change-streams/unified/change-streams.json

```diff
@@ -613,6 +614,15 @@
     },
     {
       "description": "Test new structure in ns document MUST NOT err",
+      "runOnRequirements": [
```

Comment:
This change was made in the corresponding yml file in https://github.com/mongodb/node-mongodb-native/pull/3254.  I missed the corresponding json file changes